packages/datadog_tracking_http_client/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>12.0</string>
+  <string>13.0</string>
```

Comment:
is it related to the change being made?

Reply:
Not technically, but they're automatic changes made by the version of Flutter when the application is built.